src/libServer/Server.h

```diff
@@ -434,6 +454,8 @@ class AbstractZServer : public jsonrpc::AbstractServer<AbstractZServer> {
   virtual std::string GetNodeType() = 0;
   virtual Json::Value GetDSCommittee() = 0;
   virtual std::string GetNodeState() = 0;
+  virtual Json::Value IsTxnInMemPool(const std::string& param01) = 0;
+  virtual Json::Value GetShardMembers(uint32_t shardID) = 0;
```

Comment:
For naming consistency, should we also call this `uint32_t param01`?

Reply:
done